src/VisualStudio/Xaml/Impl/CodeFixes/AddMissingNamespace/XamlAddMissingNamespacePrefixCodeFixProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Xaml.Features;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.CodeFixes.AddMissingNamespace
+{
+    [ExportCodeFixProvider(StringConstants.XamlLanguageName, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.AddMissingReference)]
+    internal class XamlAddMissingNamespacePrefixCodeFixProvider : CodeFixProvider
```

Comment:
Yes, Jason is correct. You should just put the fixer in your own assembly, and ensure that your vsixmanifest marks this assembly as a MefComponent asset:

```
<Assets>
    <Asset Type="Microsoft.VisualStudio.MefComponent" Path="<%PathToYourAssembly%>" />
</Assets>
```